pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -525,6 +525,8 @@ let
         DRM_I915_GVT_KVMGT = module;
         # Enable Hyper-V Synthetic DRM Driver
         DRM_HYPERV = whenAtLeast "5.14" module;
+        # And disable the legacy framebuffer driver
+        FB_HYPERV = no;
```

Comment:
If i understand correctly `FB_HYPERV` gets replaced by DRM_`HYPERV` and is disabled before. 
As we still have `5.4` and `5.10` kernels in nixpkgs, does it make sense to just disable it, if `FB_HYPERV` is available?

Reply:
Yes, it does. I'll update this after we confirm it works.